### PR TITLE
Translate-GameList

### PR DESCRIPTION
**Pull request by JorenC**
Hi @johnpooch 

1) I think I did it right - the strings show the right content but don't recognize the common.js. Do you know why?
2) We have these gamesstatuslabels. How do I translate these?

const gameStatusLabels = {
  [GameStatus.Started]: "Started games",
  [GameStatus.Staging]: "Open to join",
  [GameStatus.Finished]: "Finished games",
};

### CODE REVIEW

**Comment by johnpooch** [comment]
> Hi @johnpooch
> 
> 1. I think I did it right - the strings show the right content but don't recognize the common.js. Do you know why?
> 2. We have these gamesstatuslabels. How do I translate these?
> 
> const gameStatusLabels = { [GameStatus.Started]: "Started games", [GameStatus.Staging]: "Open to join", [GameStatus.Finished]: "Finished games", };

Hey @JorenC. So I think what you want to do is replace the string literals in the `gameStatusLabels` object with the translate keys:
```
const gameStatusLabels = {
  [GameStatus.Started]: tk.gameStatusLabel.started,
  [GameStatus.Staging]: tk.gameStatusLabel.staging,
  [GameStatus.Finished]: tk.gameStatusLabel.finished,
}
```

Just for context, the `GameStatus` object is the **values** of the different game statuses, i.e. the string that gets attached to the url for that particular request. This can't change. But the labels are what the user sees.

**Comment by JorenC** [comment]
@johnpooch thank you for the help, but the issue is that the labels appear to be correct, but they still don't recognize the strings in common.json, which should be correct:
image

image

File: web-app/src/pages/GameList.tsx [hunk 1]
```diff
@@ -143,7 +139,7 @@ const GameList = () => {
                           onChange={handleMasteredCheckboxChange}
                         />
                       }
-                      label={MASTERED_CHECKBOX_LABEL}
+                      label={tk.gameList.masteredGamesCheckbox.label as string}
```
Comment (johnpooch) [inline web-app/src/pages/GameList.tsx:142]: ^ Wrap line 128 `{gameStatusLabels[opt]}` with a `t()`. e.g. `{t(gameStatusLabels[opt])}`

File: web-app/src/pages/GameList.tsx [hunk 2]
```diff
@@ -112,8 +108,8 @@ const GameList = () => {
             aria-label="games filter"
             className={classes.tabs}
           >
-            <Tab value={"0"} label={ALL_GAMES_TAB_LABEL} {...a11yProps("0")} />
-            <Tab value={"1"} label={MY_GAMES_TAB_LABEL} {...a11yProps("1")} />
+            <Tab value={"0"} label={tk.gameList.allGamesTab.label} {...a11yProps("0")} />
+            <Tab value={"1"} label={tk.gameList.myGamesTab.label} {...a11yProps("1")} />
```
Comment (johnpooch) [inline web-app/src/pages/GameList.tsx:112]: Wrap these two with `t()`

**Comment by JorenC** [comment]
Closes https://github.com/zond/dipact/issues/347

**Comment by JorenC** [comment]
@johnpooch ready to pull

**Pull request merged**